OpTestConfiguration.py

```diff
@@ -117,6 +117,12 @@ def parse_args(self, argv=None):
         hostgroup.add_argument("--host-password", help="SSH password for Host")
         hostgroup.add_argument("--host-lspci", help="Known 'lspci -n -m' for host")
         hostgroup.add_argument("--host-scratch-disk", help="A block device we can erase", default="")
+        hostgroup.add_argument("--host-gateway", help="Host Gateway", default="")
+        hostgroup.add_argument("--host-submask", help="Host Subnet Mask", default="255.255.255.0")
+        hostgroup.add_argument("--host-mac", help="Host Mac address", default="")
+        hostgroup.add_argument("--host-dns", help="Host DNS", default="")
+        hostgroup.add_argument("--host-cmd", help="Shell command to run inside Host", default="")
```

Comment:
`--host-cmd` is not required as running any other commands is testcase specific, we can use `run_command` API to run any custom commands inside testcase. So please remove this one.

Reply:
@pridhiviraj this would be helpful for dynamic choose of commands to get executed inside command, planning to send a generic testcase to make use of this param..